test/Microsoft.SqlTools.ServiceLayer.Test/EditData/RowEditBaseTests.cs

```diff
@@ -178,6 +270,18 @@ public override EditUpdateCellResult SetCell(int columnId, string newValue)
             {
                 throw new NotImplementedException();
             }
+
+            public override Task ApplyChanges(DbDataReader reader)
```

Comment:
No, but they need to be there since those are the abstract methods an implementation of RowEditBase expects.